test/test_jit_fuser_te.py

```diff
@@ -950,8 +951,8 @@ def doit(x, y):
     def test_lstm(self):
         for device in self.devices:
             inputs = get_lstm_inputs(device, training=True)
-            module = self.checkScript(LSTMCellS, inputs)
-            self.assertLastGraphAllFused()
+            module = self.checkScript(LSTMCellS, inputs, extra_profile_runs=True)
+            self.assertAllFused(module.graph_for(inputs))
```

Comment:
Because we can no longer rely on last_executed_optimized_graph to query optimized graph within a DifferentiableGraph.
Use `graph_for` allows us to query the updated subgraph and display it properly: https://github.com/pytorch/pytorch/pull/52136/files#diff-6f4bcfe1c1247ea7f4287e02d4bbc0381ba54828ee99bdd8a7c207fd79441470R94